dependencyManagement/build.gradle.kts

```diff
@@ -60,7 +60,7 @@ val DEPENDENCY_SETS = listOf(
   ),
   DependencySet(
     "org.slf4j",
-    "2.0.3",
+    "1.7.36",
```

Comment:
As we are facing any issue using the latest [SLF4J API Module](https://mvnrepository.com/artifact/org.slf4j/slf4j-api). just wondering if it might have to do with this : https://www.slf4j.org/faq.html#changesInVersion200

Reply:
It is not clear what is the issue a this time, but the entry that you pointed in the faq is related to changes that were introduced in version 2.0 that are new features. I was trusting this statement:

>SLF4J 2.0.0 incorporates an optional [fluent api](https://www.slf4j.org/manual.html#fluent). Otherwise, there are no client facing API changes in 2.0.x. __For most users, upgrading to version 2.0..x should be a drop-in replacement, as long as the logging provider is updated as well.__